pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/juniperkits/JuniperKitCreationStatusTest.java

```diff
@@ -86,6 +111,29 @@ public void testStatusKitEndpointByJuniperStudyGuid() {
         }
     }
 
+    @Test
+    public void testCurrentStatusFieldAfterChanges() {
```

Comment:
Nit: `AfterChanges` may be more confusing than helpful. This appears to be the main test for the current status field, so `testCurrentStatusField` seems like an appropriate summary.